cmd/kubeadm/app/discovery/token/token_test.go

```diff
@@ -68,11 +68,14 @@ func TestRunForEndpointsAndReturnFirst(t *testing.T) {
 		},
 	}
 	for _, rt := range tests {
-		returnKubeConfig := runForEndpointsAndReturnFirst(rt.endpoints, func(endpoint string) (*clientcmdapi.Config, error) {
+		returnKubeConfig, err := runForEndpointsAndReturnFirst(rt.endpoints, func(endpoint string) (*clientcmdapi.Config, error) {
 			timeout, _ := strconv.Atoi(endpoint)
 			time.Sleep(time.Second * time.Duration(timeout))
 			return kubeconfigutil.CreateBasic(endpoint, "foo", "foo", []byte{}), nil
 		})
+		if err != nil {
+			t.Errorf("failed TestRunForEndpointsAndReturnFirst: unexpected error %v", err)
```

Comment:
Just `t.Errorf("unexpected error: %v for test %s", err, idx)`?

Reply:
The format follows the message from line 82 of the same file. I can simplify it though (as long as this does not break some test result analysis script somewhere).